test/integration/consul-container/test/gateways/http_route_test.go

```diff
@@ -260,3 +232,176 @@ func TestHTTPRouteFlattening(t *testing.T) {
 	}, checkOptions{debug: false, statusCode: service1ResponseCode, testName: "service1, v2 path with v2 hostname"})
 
 }
+
+func TestHTTPRoutePathRewrite(t *testing.T) {
+	if testing.Short() {
+		t.Skip("too slow for testing.Short")
+	}
+
+	t.Parallel()
+
+	//infrastructure set up
+	listenerPort := 6001
+	//create cluster
+	cluster := createCluster(t, listenerPort)
+	client := cluster.Agents[0].GetClient()
+	invalidServiceResponseCode := 400
+	validServiceResponseCode := 200
+	pathToRewrite := "/v1/api"
+
+	invalidService := createService(t, cluster, &libservice.ServiceOpts{
+		Name:     "invalid",
+		ID:       "invalid",
+		HTTPPort: 8080,
+		GRPCPort: 8081,
```

Comment:
If we are indeed running into port conflicts, I'm assuming these might be the issue -- that said, my understanding was that these ports were only in relation to the "pause" container, so not sure why tests would be stomping over each other.

Reply:
Did the issue wind up being port conflicts or something else?

Reply:
To be honest, the issue seems to resolve itself after I run a docker network prune, I'm not entirely sure what's causing it.